libsolutil/CDCL.cpp

```diff
@@ -230,9 +245,40 @@ std::pair<Clause, size_t> CDCL::analyze(Clause _conflictClause)
 	return {move(learntClause), backtrackLevel};
 }
 
-void CDCL::addClause(Clause _clause)
+void CDCL::addClause(const Clause& _lits)
 {
-	m_clauses.push_back(make_unique<Clause>(move(_clause)));
+	if (!ok) return;
+
+	Clause clause{_lits};
```

Comment:
This is not needed if the parameter is passed by value

Reply:
Yes, I believe it's now fixed, thanks for pointing that out!